docs/usage_general.rst.inc

```diff
@@ -404,7 +404,16 @@ Temporary files (client):
     won't be locally cached.
 
 Temporary files (server):
-    None.
+    A non-trivial (potentially 40-50 MB) amount of data will be stored on the
+    remote temp directory for each client that connects to it. For some remotes,
+    in particular QNAP NAS systems, this can fill the default (tiny!) temporary
```

Comment:
i don't think it is necessary to refer to a specific brand here.

likely all systems that mount /tmp on a tmpfs and don't have huge amounts of RAM (and a huge tmpfs size) are having this issue.

similar if /tmp is just part of the on-disk root filesystem and that one wasn't made with too much free space.